README.md

```diff
@@ -1,32 +1,50 @@
 # Bino
 
 The booking bot at your service
+### Requirements
 
-### Configure Database 
+- [Go](https://golang.org/) 1.16
+- [Docker](https://www.docker.com/)
```

Comment:
docker version can be set

Reply:
Good catch

Updated based on docker-compose file compatibility: https://docs.docker.com/compose/compose-file/